integration/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "integration-tests",
+  "version": "1.0.0",
+  "description": "Integration testing for Amazon Chime SDK React Components Library",
+  "scripts": {
+    "test": "./script/run-test",
+    "test:fast": "mocha ./test/*"
+  },
+  "license": "Apache-2.0",
+  "dependencies": {
+    "child_process": "^1.0.2",
+    "chromedriver": "^96.0.0",
+    "fs-extra": "^10.0.0",
+    "geckodriver": "^2.0.4",
```

Comment:
What if we want to run Safari locally? Is there a webkit driver?

Reply:
As we discussed earlier, we don't need to test Safari now, so I didn't add it. Do you want me to add Safari? It's simple.

Reply:
We don't need to enable it in GitHub action but we should add it so we can run it locally for testing and stuff :D

Reply:
Add the support for Safari